src/Exceptions/InvalidDirective.php

```diff
@@ -10,4 +10,9 @@ public static function notSupported(string $directive): self
     {
         return new self("The directive '${directive}' is not valid in a Feature-Policy header.");
     }
+
+    public static function unknownDirectiveType(?string $type): self
```

Comment:
added an exception method for when implementing other directives than the 'default' one.